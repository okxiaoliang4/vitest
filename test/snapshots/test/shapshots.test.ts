const println = () => {
  const message = `
export default function () {
  function Foo() {
  }

  return Foo;
}
`
  return message
}

test('non default snapshot format', () => {
  expect({ foo: ['bar'] }).toMatchInlineSnapshot(`
  Object {
    "foo": Array [
      "bar",
    ],
  }
  `)
})

test('multiline strings ', () => {
  expect(println()).toMatchSnapshot()
})
