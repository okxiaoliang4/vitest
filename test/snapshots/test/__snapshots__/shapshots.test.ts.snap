// Vitest Snapshot v1

exports[`multiline strings  1`] = `
"
export default function () {
  function Foo() {
  }

  return Foo;
}
"
`;
